### DISCUSSION TITLE
What novel begins with this phrase?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#973] [Galaxy Trivia] [answered]
> "Fasten your 💺, it's going to be a bumpy 🌃"

**Comment by peterbaccaro** [comment, marked as answer]
"Fasten your seat belts, it's going to be a bumpy night" is actually a famous quote from the film **All About Eve** (1950), directed by Joseph L. Mankiewicz and starring Bette Davis. It is not the beginning line of a novel, but rather a memorable line spoken by Bette Davis's character, Margo Channing, in the film.
